Firestore/core/src/firebase/firestore/core/event_manager.mm

```diff
@@ -109,7 +147,6 @@
   // stopListening] after an error.
   queries_.erase(found_iter);
 }
-
```

Comment:
Please restore this newline: namespaces should be separated from the code they contain.